numba/cuda/dispatcher.py

```diff
@@ -979,6 +987,28 @@ def inspect_asm(self, signature=None):
                 return {sig: overload.inspect_asm(cc)
                         for sig, overload in self.overloads.items()}
 
+    def inspect_cfg(self, signature=None):
```

Comment:
Dispatcher objects already have an `inspect_cfg()` method, which is supposed to return the CFG: https://numba.readthedocs.io/en/stable/reference/jit-compilation.html#Dispatcher.inspect_cfg - this method will need another name as it presently shadows it.

It does happen that `inspect_cfg()` is broken for CUDA dispatchers, but that's a separate issue.